### PR TITLE
Update to use default plugin dir ($HOME/.avalanchego/plugins)

### PR DESCRIPTION
**Pull request by marun**
This has been the default for avalanchego since v1.9.6.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -19,14 +19,17 @@ RUN go mod download && go mod tidy -compat=1.21
...
 
-RUN export SUBNET_EVM_COMMIT=$SUBNET_EVM_COMMIT && export CURRENT_BRANCH=$CURRENT_BRANCH && ./scripts/build.sh /build/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
+RUN export SUBNET_EVM_COMMIT=$SUBNET_EVM_COMMIT && export CURRENT_BRANCH=$CURRENT_BRANCH && ./scripts/build.sh
 
 # ============= Cleanup Stage ================
 FROM avaplatform/avalanchego:$AVALANCHE_VERSION AS builtImage
 
 # Copy the evm binary into the correct location in the container
-COPY --from=builder /build/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy /avalanchego/build/plugins/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
+COPY --from=builder /build/build/subnet-evm /avalanchego/build/plugins/srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
```
Comment (darioush) [inline Dockerfile:35]: assuming that `/build/build` is intentional

**Review by darioush — DISMISSED** [review]

**Review by StephenButtolph — APPROVED** [review]
❤️

**Review by ceyonur — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (ceyonur) [reply Dockerfile:35]: I don't think there is. I think it looks good but could you spin up a Docker img with Docker file to double check this?

Comment (ceyonur) [reply Dockerfile:35]: I run it, seems good.

**Review by ceyonur — APPROVED** [review]

**Review by darioush — APPROVED** [review]

**Pull request merged**